File: unittest/scripts/auto/js_devapi/scripts/mysqlx_rowresult_help.js
//@ Initialization
shell.connect(__uripwd + '/mysql');
var result = db.user.select().execute();
session.close();

//@ RowResult help
result.help();

//@ RowResult help, \? [USE:RowResult help]
\? RowResult

//@ Help on affectedItemsCount
result.help('affectedItemsCount');

//@ Help on affectedItemsCount, \? [USE:Help on affectedItemsCount]
\? RowResult.affectedItemsCount

//@ Help on columnCount
result.help('columnCount');

//@ Help on columnCount, \? [USE:Help on columnCount]
\? RowResult.columnCount

//@ Help on columnNames
result.help('columnNames');

//@ Help on columnNames, \? [USE:Help on columnNames]
\? RowResult.columnNames

//@ Help on columns
result.help('columns');

//@ Help on columns, \? [USE:Help on columns]
\? RowResult.columns

//@ Help on executionTime
result.help('executionTime');

//@ Help on executionTime, \? [USE:Help on executionTime]
\? RowResult.executionTime

//@ Help on warningCount
result.help('warningCount');

//@ Help on warningCount, \? [USE:Help on warningCount]
\? RowResult.warningCount

//@ Help on warnings
result.help('warnings');

//@ Help on warnings, \? [USE:Help on warnings]
\? RowResult.warnings

//@ Help on warningsCount
result.help('warningsCount');

//@ Help on warningsCount, \? [USE:Help on warningsCount]
\? RowResult.warningsCount

//@ Help on fetchAll
result.help('fetchAll');

//@ Help on fetchAll, \? [USE:Help on fetchAll]
\? RowResult.fetchAll

//@ Help on fetchOne
result.help('fetchOne');

//@ Help on fetchOne, \? [USE:Help on fetchOne]
\? RowResult.fetchOne

//@ Help on getAffectedItemsCount
result.help('getAffectedItemsCount');

//@ Help on getAffectedItemsCount, \? [USE:Help on getAffectedItemsCount]
\? RowResult.getAffectedItemsCount

//@ Help on getColumnCount
result.help('getColumnCount');

//@ Help on getColumnCount, \? [USE:Help on getColumnCount]
\? RowResult.getColumnCount

//@ Help on getColumnNames
result.help('getColumnNames');

//@ Help on getColumnNames, \? [USE:Help on getColumnNames]
\? RowResult.getColumnNames

//@ Help on getColumns
result.help('getColumns');

//@ Help on getColumns, \? [USE:Help on getColumns]
\? RowResult.getColumns

//@ Help on getExecutionTime
result.help('getExecutionTime');

//@ Help on getExecutionTime, \? [USE:Help on getExecutionTime]
\? RowResult.getExecutionTime

//@ Help on getWarningCount
result.help('getWarningCount');

//@ Help on getWarningCount, \? [USE:Help on getWarningCount]
\? RowResult.getWarningCount

//@ Help on getWarnings
result.help('getWarnings');

//@ Help on getWarnings, \? [USE:Help on getWarnings]
\? RowResult.getWarnings

//@ Help on getWarningsCount
result.help('getWarningsCount');

//@ Help on getWarningsCount, \? [USE:Help on getWarningsCount]
\? RowResult.getWarningsCount

//@ Help on help
result.help('help');

//@ Help on help, \? [USE:Help on help]
\? RowResult.help

//@ Finalization
session.close();

File: unittest/scripts/auto/py_devapi/scripts/mysqlx_rowresult_help.py
shell.connect(__uripwd + '/mysql')
rowresult = db.user.select().execute()
session.close()

#@ rowresult
rowresult.help()

#@ global ? for RowResult[USE:rowresult]
\? RowResult

#@ global help for RowResult[USE:rowresult]
\help RowResult

#@ rowresult.affected_items_count
rowresult.help('affected_items_count')

#@ global ? for affected_items_count[USE:rowresult.affected_items_count]
\? RowResult.affected_items_count

#@ global help for affected_items_count[USE:rowresult.affected_items_count]
\help RowResult.affected_items_count

#@ rowresult.column_count
rowresult.help('column_count')

#@ global ? for column_count[USE:rowresult.column_count]
\? RowResult.column_count

#@ global help for column_count[USE:rowresult.column_count]
\help RowResult.column_count

#@ rowresult.column_names
rowresult.help('column_names')

#@ global ? for column_names[USE:rowresult.column_names]
\? RowResult.column_names

#@ global help for column_names[USE:rowresult.column_names]
\help RowResult.column_names

#@ rowresult.columns
rowresult.help('columns')

#@ global ? for columns[USE:rowresult.columns]
\? RowResult.columns

#@ global help for columns[USE:rowresult.columns]
\help RowResult.columns

#@ rowresult.execution_time
rowresult.help('execution_time')

#@ global ? for execution_time[USE:rowresult.execution_time]
\? RowResult.execution_time

#@ global help for execution_time[USE:rowresult.execution_time]
\help RowResult.execution_time

#@ rowresult.fetch_all
rowresult.help('fetch_all')

#@ global ? for fetch_all[USE:rowresult.fetch_all]
\? RowResult.fetch_all

#@ global help for fetch_all[USE:rowresult.fetch_all]
\help RowResult.fetch_all

#@ rowresult.fetch_one
rowresult.help('fetch_one')

#@ global ? for fetch_one[USE:rowresult.fetch_one]
\? RowResult.fetch_one

#@ global help for fetch_one[USE:rowresult.fetch_one]
\help RowResult.fetch_one

#@ rowresult.get_affected_items_count
rowresult.help('get_affected_items_count')

#@ global ? for get_affected_items_count[USE:rowresult.get_affected_items_count]
\? RowResult.get_affected_items_count

#@ global help for get_affected_items_count[USE:rowresult.get_affected_items_count]
\help RowResult.get_affected_items_count

#@ rowresult.get_column_count
rowresult.help('get_column_count')

#@ global ? for get_column_count[USE:rowresult.get_column_count]
\? RowResult.get_column_count

#@ global help for get_column_count[USE:rowresult.get_column_count]
\help RowResult.get_column_count

#@ rowresult.get_column_names
rowresult.help('get_column_names')

#@ global ? for get_column_names[USE:rowresult.get_column_names]
\? RowResult.get_column_names

#@ global help for get_column_names[USE:rowresult.get_column_names]
\help RowResult.get_column_names

#@ rowresult.get_columns
rowresult.help('get_columns')

#@ global ? for get_columns[USE:rowresult.get_columns]
\? RowResult.get_columns

#@ global help for get_columns[USE:rowresult.get_columns]
\help RowResult.get_columns

#@ rowresult.get_execution_time
rowresult.help('get_execution_time')

#@ global ? for get_execution_time[USE:rowresult.get_execution_time]
\? RowResult.get_execution_time

#@ global help for get_execution_time[USE:rowresult.get_execution_time]
\help RowResult.get_execution_time

#@ rowresult.get_warning_count
rowresult.help('get_warning_count')

#@ global ? for get_warning_count[USE:rowresult.get_warning_count]
\? RowResult.get_warning_count

#@ global help for get_warning_count[USE:rowresult.get_warning_count]
\help RowResult.get_warning_count

#@ rowresult.get_warnings
rowresult.help('get_warnings')

#@ global ? for get_warnings[USE:rowresult.get_warnings]
\? RowResult.get_warnings

#@ global help for get_warnings[USE:rowresult.get_warnings]
\help RowResult.get_warnings

#@ rowresult.get_warnings_count
rowresult.help('get_warnings_count')

#@ global ? for get_warnings_count[USE:rowresult.get_warnings_count]
\? RowResult.get_warnings_count

#@ global help for get_warnings_count[USE:rowresult.get_warnings_count]
\help RowResult.get_warnings_count

#@ rowresult.help
rowresult.help('help')

#@ global ? for help[USE:rowresult.help]
\? RowResult.help

#@ global help for help[USE:rowresult.help]
\help RowResult.help

#@ rowresult.warning_count
rowresult.help('warning_count')

#@ global ? for warning_count[USE:rowresult.warning_count]
\? RowResult.warning_count

#@ global help for warning_count[USE:rowresult.warning_count]
\help RowResult.warning_count

#@ rowresult.warnings
rowresult.help('warnings')

#@ global ? for warnings[USE:rowresult.warnings]
\? RowResult.warnings

#@ global help for warnings[USE:rowresult.warnings]
\help RowResult.warnings

#@ rowresult.warnings_count
rowresult.help('warnings_count')

#@ global ? for warnings_count[USE:rowresult.warnings_count]
\? RowResult.warnings_count

#@ global help for warnings_count[USE:rowresult.warnings_count]
\help RowResult.warnings_count
